mtp_api/assets/core/js/satisfaction-results.js

```diff
@@ -48,27 +53,112 @@ django.jQuery(function($) {
         }
       },
       backgroundColor: '#f9f9f9',
-      colors: ['#79aec8']
+      colors: colours
     });
 
     google.visualization.events.addListener(chart, 'ready', function() {
       var $svg = $chart.find('svg'),
+        svgNamespace = $svg[0].namespaceURI,
         cli = chart.getChartLayoutInterface(),
         legendBounds = cli.getBoundingBox('legend'),
-        $title;
+        chartBounds = cli.getChartAreaBoundingBox(),
+        significantWidth = chartBounds.width * 5 / 6, // last option is disregarded
```

Comment:
it's deliberate, the average response is somewhere between "strongly disagree" and "strongly agree", i.e. "not applicable" (the 6th response) is disregarded for the width calculation.